imports/plugins/custom/products-review/client/components/Reviews.js

```diff
@@ -0,0 +1,264 @@
+import React, { Component } from "react";
+import ReactStars from "react-stars";
+import { Meteor } from "meteor/meteor";
+import "../stylesheet/style.css";
+import PropTypes from "prop-types";
+import moment from "moment";
+import Pagination from "react-js-pagination";
+// import toastrOption from "../../utils/toastrOption";
+/**
+ * @class Review
+ *
+ * @classdesc user post review
+ *
+ */
+class Reviews extends Component {
+  /**
+   * constructor - contains the constructor
+   *
+   * @param  {object} props the properties of the class component
+   *
+   * @return {void} no return or void
+   *
+   */
+  constructor(props) {
+    super(props);
+    this.state = {
+      review: "",
+      rating: 0,
+      username: "",
+      disableBtn: true,
+      reviews: "",
+      totalCount: 0,
+      totalRating: 0,
+      activePage: 1,
+      paginatedReviews: []
+    };
+    this.onChange = this.onChange.bind(this);
+    this.ratingChanged = this.ratingChanged.bind(this);
+    this.addReview = this.addReview.bind(this);
+    this.reviewList = this.reviewList.bind(this);
+    this.handlePageChange = this.handlePageChange.bind(this);
+  }
+  /**
+ *
+ * @returns { object } updated state
+ * @memberof Ratings
+*/
+  componentDidMount() {
+    const destination = this.props.product._id;
```

Comment:
A field/name that has an underscore as a prefix signifies a private datatype in JS by convention. So, you shouldn't access it directly from the client. I suggest you write a method/publication in the server that takes in a product object as the param and returns the id.

Reply:
Thanks @bekomay26 . However, this follows convention demonstrated in Reaction Commerce documentation.

Reply:
I agree with @ikenjoku, it is accepted in Reaction Commerce Community